fastpin.h

```diff
@@ -236,6 +238,21 @@ template<uint8_t PIN> class FastPin {
 
 #endif
 
+template<uint8_t PIN> class FastOutputPin : public FastPin<PIN> {
```

Comment:
I don't think it makes sense to have these - especially since setInput can be called on a FastOutputPin and vice versa.  This doesn't really save a whole lot.

Reply:
I know, but those also exist for the normal class(non template) variants. And I had a use case where I was missing those for templates. Still should remove them?

Reply:
Ah - not sure why those are still in there, they shouldn't be - not least of all because they are mutable, so they're a bit of a lie : ) -- i'll clean that up at some point, in the meantime, I don't want to encourage spreading their use by adding more variants.  

The new methods being made available across all the platforms is actually the more important one for me, though.